content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: 'Plans and quotas'
+metaTitle: 'Plans and quotas'
+metaDescription: 'The Prisma Data Platform provides a set of monthly plans that include quotas for the number of collaborators and environments per project, as well as a quota for the usage of the Prisma Data Proxy (in CPU milliseconds).'
+---
+
+<TopBlock>
+
+The Prisma Data Platform provides a set of monthly plans that include quotas for:
+
+- number of collaborators per project
+- number of environments per project
+- Prisma Data Proxy usage (in CPU milliseconds) per project
+
+</TopBlock>
+
+## About plans and quotas
+
+The important thing to know about plans in the Prisma Data Platform is that a single plan is attached to a single project.
+
+<Admonition type="alert">
+
+**Account-wide plans**<br /><br />
```

Comment:
Do we intentionally want to point out what we *do not* have, or would it make sense to instead point out that our paid plans are *per project*?

It'd mean the same, but focus on what we do have vs. what we do not.

We might want to do this intentionally, which is perfectly fine. I'm asking because as I opened the rendered draft page, the red alert stuck out so much, that the first thing I processed was something we do not offer. In my head, that didn't put us in the best light. It might be just me though.

Reply:
Thanks, that's a good point, @petradonka. When I created this, I wanted to make sure from the beginning that people don't get confused by our notion of what a plan is. So the purpose here was clarity. The execution is rather screaming, I agree.

I've rewritten this intro text (hopefully it's a little bit friendlier) and have pushed the alert to the bottom and changed that to a note-type admonition.

Reply:
Looks great! 👌 ✨